docs/app/[...slug]/page.tsx

```diff
@@ -26,6 +27,22 @@ export async function generateMetadata({
     ? {
         title: page.title,
         description: page.caption,
+        applicationName: 'Marigold Design System',
+        appleWebApp: {
+          title: 'Marigold Design System',
+        },
+        metadataBase: new URL(baseUrl),
+        openGraph: {
+          siteName: 'Marigold Design System',
+          title: page.title,
+          description: page.caption,
+          images: `${baseUrl}/og.png?title=${encodeURIComponent(page.title)}`,
```

Comment:
Is there a reason why the directory is named og.png? So why no subdirectory og/png?

Reply:
because the URL is then an image called "og.png". this will automatically cause browsers to infer the correct headers etc